src/assign.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * assign
+ * Non-destructively assign one object into another
+ */
+
+export default function assign (a, b) {
+  return { ...a, ...b }
```

Comment:
Yeah! I love it. I would have done it more inline within the code but I thought this was a bit clearer